src/buildercore/trop.py

```diff
@@ -49,6 +51,8 @@
 
 KEYPAIR = "KeyName"
 
+FASTLY_GLOBAL = 'nonssl.global.fastly.net.'
```

Comment:
The other options is a hostname for just EU/US edge nodes, which costs less but has worse performance outside of EU/US.